meson.build

```diff
@@ -67,7 +67,7 @@ if get_option('nls') and cc.has_header('libintl.h')
     if cc.has_function('dgettext', dependencies : libintl)
       libintl_deps += libintl
       if ['darwin', 'ios'].contains(host_system)
-        appleframeworks = dependency('appleframeworks', modules : 'foundation')
+        appleframeworks = dependency('appleframeworks', modules : 'Foundation')
```

Comment:
I suspect we might only need CoreFoundation, as the libintl only checks CF* functions:
https://git.savannah.gnu.org/gitweb/?p=gnulib.git;a=blob;f=m4/intlmacosx.m4;h=3141bf8c5e4c949f03d4757701ca9c5fc3c6c9a1;hb=HEAD